css/parser/tests/fixture/vercel/001/output.json

```diff
@@ -0,0 +1,204 @@
+        "span": {
+          "start": 8,
+          "end": 25,
+          "ctxt": 0
+        },
+        "name": {
+          "type": "Text",
+          "span": {
+            "start": 8,
+            "end": 17,
+            "ctxt": 0
+          },
+          "value": "min-width"
+        },
+        "values": [
+          {
+            "type": "UnitValue",
+            "span": {
+              "start": 19,
+              "end": 24,
+              "ctxt": 0
+            },
+            "value": {
+              "type": "Number",
+              "span": {
+                "start": 19,
+                "end": 22,
+                "ctxt": 0
+              },
+              "value": 935.0
+            },
+            "unit": {
+              "span": {
+                "start": 22,
+                "end": 24,
+                "ctxt": 0
+              },
+              "kind": null
+            }
+          }
+        ],
+        "important": null
+      },
+      "rules": [
+        {
+          "type": "StyleRule",
+          "span": {
+            "start": 32,
+            "end": 85,
+            "ctxt": 0
+          },
+          "selectors": [
+            {
+              "type": "ComplexSelector",
+              "span": {
+                "start": 32,
+                "end": 52,
+                "ctxt": 0
+              },
+              "selectors": [
+                {
+                  "type": "CompoundSelector",
+                  "span": {
+                    "start": 32,
+                    "end": 52,
+                    "ctxt": 0
+                  },
+                  "hasNestPrefix": false,
+                  "combinator": null,
+                  "typeSelector": null,
+                  "subclassSelectors": [
+                    {
+                      "type": "ClassSelector",
+                      "span": {
+                        "start": 32,
+                        "end": 38,
+                        "ctxt": 0
+                      },
+                      "text": {
+                        "type": "Text",
+                        "span": {
+                          "start": 33,
+                          "end": 38,
+                          "ctxt": 0
+                        },
+                        "value": "video"
+                      }
+                    },
+                    {
+                      "type": "PseudoSelector",
+                      "span": {
+                        "start": 38,
+                        "end": 52,
+                        "ctxt": 0
+                      },
+                      "isElement": false,
+                      "name": {
+                        "type": "Text",
+                        "span": {
+                          "start": 39,
+                          "end": 48,
+                          "ctxt": 0
+                        },
+                        "value": "nth-child"
+                      },
+                      "args": {
```

Comment:
What do you mean?
Ah, I think I know what you are saying. This test file is in `tests/fixture` and it means it's a valid css file.

Title of the PR is about the change I intended to make, but while debugging the actual bug, I added this file to valid file list because I want to sure that this is parsed correctly. (This file was a part of errornous input)